torchgeo/transforms/indices.py

```diff
@@ -116,6 +131,45 @@ def forward(self, sample: Dict[str, Tensor]) -> Dict[str, Tensor]:
         return sample
 
 
+class AppendNBR(Module):  # type: ignore[misc,name-defined]
+    """Normalized Burn Ratio (NBR).
+    .. versionadded:: 0.2.0
```

Comment:
I think you still want an empty line between these. The reason that black/flake8 were failing was because the line contained spaces and wasn't actually empty, which violates PEP-8. Btw, you can always run `black .` and have black auto-format all of your code so you don't need to deal with any of this. See https://torchgeo.readthedocs.io/en/stable/user/contributing.html#linters for details